### PR TITLE
nsc-events-nextjs_7_376_401-unauthorized-for-create-event-page

### PR DESCRIPTION
**Pull request by kaimanasse15**
This is the 401 unauthorized updated.
Issue: #376

The last PR has been closed. Please update the code with this

### CODE REVIEW

**Review by heosman — COMMENTED** [review]
In your other PR for the Create Eevent page that was merged, It would direct us to a page with the text "You are not authorized to access this page." Could you keep that along with the "401 Unauthorized" text above it? I think this makes it easier for the user to understand the issue. It would look like this:

401 unauthorized error

**Comment by kaimanasse15** [comment]
Sure! I will fix it! Thanks, @heosman!

File: components/UnauthorizedPages.tsx
Comment (kaimanasse15) [inline components/UnauthorizedPages.tsx:1]: Please update this code:
import Box from "@mui/material/Box";
import Typography from "@mui/material/Typography";

const UnauthorizedPages: React.FC = () => {
  return (
    <Box sx={{ display: 'flex', justifyContent: 'center', alignItems: 'center', height: '80vh', flexDirection: 'column' }}>
      <Typography variant="h5" component="h2">
        401 Unauthorized
      </Typography>
      <Typography variant="h5" component="h2" sx={{ maxWidth: 500 }}>
        You are not authorized to access this page
      </Typography>
    </Box>
  );
};

export default UnauthorizedPages;

**Review by taylorpapke — APPROVED** [review]
UnauthorizedPages.tsx component is a nice touch since it can be reused for other pages. It looks like it is working: 
Screenshot 2024-05-28 at 8 48 44 PM

I also agree with what @hanan mentioned as far as including what she has in her screenshot. Approving in advance of those changes being made.

**Review by Robel-003 — COMMENTED** [review]
Working for me as well. I get the 401 unauthorized page when attempting to visit the create event page. 

Screenshot 2024-05-29 at 3 21 35 PM

**Comment by heosman** [comment]
@kaimanasse15 I noticed that you were able to update the text for the Unauthorized message in your other PR. Have you done the same for this PR? As soon as those changes are made, this PR can be merged. Also, there seems to be unpushed commits on your branch.

File: app/create-event/page.tsx
Comment (gitbiruk2010) [inline app/create-event/page.tsx:1]: This is good and makes it easier to understand for the user about what is happening and just tells them they do not have access and it's not another issue. good job.

**Comment by kaimanasse15** [comment]
> @kaimanasse15 I noticed that you were able to update the text for the Unauthorized message in your other PR. Have you done the same for this PR? As soon as those changes are made, this PR will be ready to be merged. Also, there seems to be unpushed commits on your branch.

Somehow, I am unable to commit it again. I think maybe use the other PR to merge on main. Thank you, @heosman

**Comment by heosman** [comment]
> > @kaimanasse15 I noticed that you were able to update the text for the Unauthorized message in your other PR. Have you done the same for this PR? As soon as those changes are made, this PR will be ready to be merged. Also, there seems to be unpushed commits on your branch.
> 
> Somehow, I am unable to commit it again. I think maybe use the other PR to merge on main. Thank you, @heosman

I'll close this PR then.

**Pull request closed without merging**
